docs/lib/examples/Badge.js

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import { Badge } from 'reactstrap';
 
-export default class Example extends React.Component {
-  render() {
-    return (
-      <div>
-        <h1>Heading <Badge color="secondary">New</Badge></h1>
-        <h2>Heading <Badge color="secondary">New</Badge></h2>
-        <h3>Heading <Badge color="secondary">New</Badge></h3>
-        <h4>Heading <Badge color="secondary">New</Badge></h4>
-        <h5>Heading <Badge color="secondary">New</Badge></h5>
-        <h6>Heading <Badge color="secondary">New</Badge></h6>
-      </div>
-    );
-  }
+const Example = (props) => {
+  return (
```

Comment:
Since you are doing a functional components you can always just directly return the `jsx` here rather than having a return method. Just a recommendation for all of these refactors but otherwise LGTM.
```jsx
const Example = props => (<div>stuff</div>)
```

Reply:
the reason why I’m not implicitly returning is because I wanted to make the docs as explicit as possible, to avoid confusion for new JS developers (hence why I’m also including “props” as an argument for components that don’t use it). 

I should’ve went more explicit with the syntax and used “function” declarations instead of arrow function declarations but I wanted the doc examples to be consistent and I really didn’t feel like refactoring the other function component examples 😛the diff for this PR is really big already.

Reply:
Sounds good. Agreed she’s a big one 🎣